src/App.tsx

```diff
@@ -0,0 +1,48 @@
+import { lazy, Suspense } from "react";
+import { ToastContainer } from "react-toastify";
+import { useSelector } from "react-redux";
+import Header from "./components/Header/Header";
+import Preloader from "./components/Preloader/Preloader";
```

Comment:
Weird imports, i suggest using index.ts file for reexport so you line would look like
import Preloader from "./components/Preloader";
Or even like this
import { Preloader, Header } from "./components";
Also would make sense to make alias for global imports like @/components that you can use in any file in any location.